.gitignore

```diff
@@ -1,37 +1,24 @@
-# See https://help.github.com/ignore-files/ for more about ignoring files.
-
-# dependencies
-/node_modules
-
-# testing
-/coverage
-
-# production
-/build
-
-# release
-/release
-
-# jetbrains
-/.idea
-
-# vscode
-/.vscode
-
-# misc
-.DS_Store
-.env.local
-.env.development.local
-.env.test.local
-.env.production.local
-
-# yarn lock file
-yarn.lock

```

Comment:
I see we use "yarn install" in .github/workflow, and I think yarn is more reliable and fast, so to make it consistent, I recommend to using yarn as our major package manager.

So we keep yarn.lock.